specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json

```diff
@@ -121,29 +117,25 @@
         }
       },
       "put": {
```

Comment:
PUT configurations API signature incorrect. the name segment is missing. If its singleton, then call it "default"
SO it will be PUT /subscriptions/{id}/Microsoft.Advisor/configurations/{name} and the name will be an enum and  support just one value which will be "default"